ios/RNBLEPrinter.m

```diff
@@ -8,17 +8,12 @@
 #import <Foundation/Foundation.h>
 
 #import "RNBLEPrinter.h"
-#import "PrinterSDK.h"
+#import "adapters/epson/BLEPrinterEpsonAdapter.h"
+#import "adapters/general/BLEPrinterGenericAdapter.h"
```

Comment:
you mean `adapters/generic/BLEPrinterGenericAdapter.h`

Reply:
generic or general?
Ok. Will change them to generic.